<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLASHQuantTabWidget</class>
 <widget class="QTabWidget" name="FLASHQuantTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>394</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <property name="elideMode">
   <enum>Qt::ElideNone</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabsClosable">
   <bool>false</bool>
  </property>
  <property name="tabBarAutoHide">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="tab_lcms_files">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="title">
    <string>LC-MS files</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="toolTip">
       <string>Hint: only non-split mzML files are supported.</string>
      </property>
      <property name="title">
       <string>mzML input files</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Drag'n'drop files or use the 'Add ...' button to add files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="InputFileList" name="input_mzMLs" native="true">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_run">
   <attribute name="title">
    <string>Run FLASHQuant</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2,9">
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Output folder</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Select a directory to place the results into. Leaving this empty will place results in your $HOME/FLASHQuantOut directory.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_outputdirectory">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="OutputDirectory" name="out_dir" native="true"/>
         </item>
         <item>
          <widget class="QPushButton" name="open_output_directory">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_optionalout" stretch="3,1">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Select optional deconvoluted output file formats. </string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QCheckBox" name="checkbox_featxml">
           <property name="text">
            <string>featureXML</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Output consensus feature group</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkbox_consensus">
           <property name="text">
            <string>consensus output (*.tsv)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Indication to distinguish replicates. &lt;/p&gt;&lt;p&gt;If not given, all input LC-MS files will be used to report Consensus feature groups.&lt;/p&gt;&lt;p&gt;Currently, replicates are required to share the same file name except for the last few characters. For example, with the file names of File_name_R1.tsv and File_name_R2.tsv, please input &amp;quot;_R&amp;quot; for the indication. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Prefix of replicate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="consensus_name_edit">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Indication to distinguish replicates. &lt;/p&gt;&lt;p&gt;If not given, all input LC-MS files will be used to report Consensus feature groups.&lt;/p&gt;&lt;p&gt;Currently, replicates are required to share the same file name except for the last few characters. For example, with the file names of File_name_R1.tsv and File_name_R2.tsv, please input &amp;quot;_R&amp;quot; for the indication. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>i.e) _R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="consensus_names_check">
           <property name="text">
            <string>check</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Run FLASHQuant</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="ParamEditor" name="list_editor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edit the most important parameters directly below.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="run_fdq">
         <property name="text">
          <string>run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_log">
   <attribute name="title">
    <string>Log</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="log_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputFileList</class>
   <extends>QWidget</extends>
   <header>OpenMS/VISUAL/InputFileList.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParamEditor</class>
   <extends>QWidget</extends>
   <header>OpenMS/VISUAL/ParamEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputDirectory</class>
   <extends>QWidget</extends>
   <header>OpenMS/VISUAL/OutputDirectory.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
